### PR TITLE
Updated with 3rd Party RPC Provider Section

### PR DESCRIPTION
**Pull request by j0sh7**
Added 3rd Party RPC Section and Ankr Avalanche RPC's

### CODE REVIEW

File: docs/apis/avalanchego/public-api-server.md [hunk 1]
```diff
@@ -47,7 +47,18 @@ avalanche.setRequestConfig("withCredentials", true);
 
 ## Availability
 
-Usage of public API nodes is free and available to everyone without any authentication or authorization. Rate limiting is present, but many of the API calls are cached, and the rate limits are quite high. If your application is running up against the limits, please [contact us](https://chat.avalabs.org).
+Usage of public API nodes is free and available to everyone without any authentication or authorization. Rate limiting is present, but many of the API calls are cached, and the rate limits are quite high. If your application is running up against the limits, please [contact us](https://chat.avalabs.org) or try using a 3rd party RPC provider. 
+
+## 3rd Party Public RPC Providers
```
Comment (yulin-dong) [inline docs/apis/avalanchego/public-api-server.md:52]: Please move this section to the bottom of this document.

File: docs/apis/avalanchego/public-api-server.md [hunk 2]
```diff
@@ -47,7 +47,18 @@ avalanche.setRequestConfig("withCredentials", true);
...
+
+### Ankr
+* For C-Chain API, the URL is `https://rpc.ankr.com/avalanche`.
+* For X-Chain API, the URL is `https://rpc.ankr.com/avalanche-x`.
+* For P-Chain API, the URL is `https://rpc.ankr.com/avalanche-p`.
+* For Fuji Testnet API, the URL is `https://rpc.ankr.com/avalanche_fuji`.
+
+Note: soft limited to 1 million daily requests per IP or referring domain. Includes archive data. Batch calls limited to 1000.
```
Comment (yulin-dong) [inline docs/apis/avalanchego/public-api-server.md:60]: Please add information on where developer can reach out if they have questions on the Ankr's server availability.

Comment (j0sh7) [reply docs/apis/avalanchego/public-api-server.md:60]: Done in latest commit.

File: docs/apis/avalanchego/public-api-server.md (hunk 1 above)
Comment (j0sh7) [reply docs/apis/avalanchego/public-api-server.md:52]: Done in latest commit.

**Pull request closed without merging**